src/webgpu/web_platform/external_texture/video.spec.ts

```diff
@@ -138,6 +148,28 @@ function createExternalTextureSamplingTestBindGroup(
   return bindGroup;
 }
 
+function getVideoElementAndInfo(
+  t: GPUTest,
+  sourceType: 'VideoElement' | 'VideoFrame',
+  videoName: VideoName
+): [videoElement: HTMLVideoElement, videoInfo: VideoInfo] {
```

Comment:
µNit: Slightly prefer returning objects over tuples when using them this way, since it lets you name the members of the object.